build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    let nightly = std::env::var_os("CARGO_FEATURE_NIGHTLY").is_some();
+    let has_stable_alloc = || autocfg::new().probe_rustc_version(1, 36);
```

Comment:
Is the nightly check necessary? Won't nightly always have a version number above 1.36?

Reply:
Not if you compile with an old nightly?

Reply:
Fair enough